api/src/main/java/org/pmiops/workbench/initialcredits/InitialCreditsExpirationServiceImpl.java

```diff
@@ -93,9 +93,12 @@ && haveCreditsExpired(user)
                       ws.getBillingAccountName(), workbenchConfigProvider.get()))
           .filter(DbWorkspace::isActive)
           .filter(ws -> ws.getBillingStatus().equals(BillingStatus.ACTIVE))
```

Comment:
because this field is no longer the source of truth, I would prefer to stop reading it in *all cases* from this point onward - so: no calls to `getBillingStatus()` at all (except mappers and similar)